config/routes.rb

```diff
@@ -18,21 +18,20 @@
     get "/test-search/search/opensearch" => "search#opensearch"
   end
 
-  # Routes for the for Transition/Brexit Checker
-  get "/transition-check/results" => "brexit_checker#results", as: :transition_checker_results
-  get "/transition-check/questions" => "brexit_checker#show", as: :transition_checker_questions
-  get "/transition-check/email-signup" => "brexit_checker#email_signup", as: :transition_checker_email_signup
-  post "/transition-check/email-signup" => "brexit_checker#confirm_email_signup", as: :transition_checker_confirm_email_signup
+  scope "/transition-check" do
+    get "/results" => "brexit_checker#results", as: :transition_checker_results
+    get "/questions" => "brexit_checker#show", as: :transition_checker_questions
+    get "/email-signup" => "brexit_checker#email_signup", as: :transition_checker_email_signup
+    post "/email-signup" => "brexit_checker#confirm_email_signup", as: :transition_checker_confirm_email_signup
+    get "/save-your-results" => "brexit_checker#save_results", as: :transition_checker_save_results
+  end
 
   # Transition/Brexit checker email signup routes
   get "/email/subscriptions/new", to: proc { [200, {}, [""]] }, as: :email_alert_frontend_signup
 
   get "/*slug/email-signup" => "email_alert_subscriptions#new", as: :new_email_alert_subscriptions
   post "/*slug/email-signup" => "email_alert_subscriptions#create", as: :email_alert_subscriptions
 
-  # GOV.UK Accounts discovery routes
```

Comment:
Potentially.... ditto with this one...
As if the checker might outlive the account discovery... this could be worth highlighting to anyone that might want to unpick the code later?

Reply:
I'm hoping we should be able to unpick accounts through the pull request and commit history, should we need to remove them from the checker

Reply:
fair, commits over comments, you're right!